src/pager.js

```diff
@@ -38,6 +39,9 @@ export class Pager {
   pageChanged(newValue, oldValue) {
     if (newValue !== oldValue) {
       this.goToPage(newValue);
+	  if (this.onPageChanged !== null) {
```

Comment:
`if (typeof this.onPageChanged === 'function') {}` is probably a better idea.

Also, add two spaces in front of the if (indentation is wrong now) and add a newline before the if.

But reading your issue, I'm not sure this solves your problem. You might want to wait for it to finish loading data. If not, this PR is fine :)